packages/suite/src/constants/suite/anchors.ts

```diff
@@ -1,6 +1,7 @@
 export const enum SettingsAnchor {
     Language = '@general-settings/language',
     Fiat = '@general-settings/fiat',
+    BitcoinUnits = '@general-settings/btc-units',
```

Comment:
```suggestion
    AmountUnit = '@general-settings/amount-unit',
```
let's keep consistent naming